script/create-practice-repos

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env bash
+#
+# Create practice repos
+
+#################################################################
+# NOTE: You must have a personal access token (PAT)             #
+# saved to your environment variables to use this script.       #
+# We recommend a dedicated service account (e.g. githubteacher) #
+#################################################################
+
+# shellcheck source=script/shared_functions
+source ./shared_functions
+
+# shell variables
+collab_repo=$1
+template_name=$2
+practice_repo_name=$3
+org_url="https://$ROOT_URL/$CLASS_ORG"
+org_endpoint="https://$INSTANCE_URL/repos/$CLASS_ORG"
+template_url="https://github.com/githubtraining/$template_name"
```

Comment:
Might be a very picky question, but why are we hard coding github.com and githubtraining.  For the regular .com class wouldn't $ROOT_URL be github.com and $CLASS_ORG be githubtraining in which case the URLs for .com and GHES would be the same.  The only one with a real difference is pages since you're dealing with the /pages vs .github.io.

Reply:
This is so we can use the public template as a fallback: https://github.com/githubtraining/training-manual/pull/279/files/3112388e21216250b87348504ebf4fb3040cbce4#r578082025